packages/widget/src/components/common/dropdown/dropdown.ts

```diff
@@ -105,7 +115,22 @@ export class Dropdown extends BaseComponent {
     );
   }
 
-  _renderOptions(): Generator<unknown, void> {
+  private renderOptionContent({
+    customOptionHtml,
+    name,
+    icon
+  }: DropdownOption): HTMLTemplateResult {
+    return when(
+      customOptionHtml,
+      () => customOptionHtml,
+      () => html`
+        ${icon || ''}
+        <span class="optionName">${capitalize(name)}</span>
+      `
+    ) as HTMLTemplateResult;
```

Comment:
does it ask you to cast the type here? I mean the linter?

Reply:
It's actually could be undefined as well